### ISSUE TITLE
Crash in `MPV().show_text(..)`

### ISSUE THREAD

**Issue by kpj** [jaseg/python-mpv#69]
Consider the following code:
```python
import mpv

mpv = mpv.MPV()
mpv.play('test.webm')
mpv.show_text('Hello MPV!')
mpv.wait_for_playback()
```

I would expect it to display `Hello MPV!`. Instead I get the following error:
```bash
Traceback (most recent call last):
  File "test.py", line 5, in <module>
    mpv.show_text('Hello MPV!')
  File "/site-packages/mpv.py", line 747, in show_text
    self.command('show_text', string, duration, level)
  File "/site-packages/mpv.py", line 615, in command
    _mpv_command(self.handle, (c_char_p*len(args))(*args))
  File "/site-packages/mpv.py", line 102, in raise_for_ec
    raise ex(ec, *args)
ValueError: ('Invalid value for mpv parameter', -4, (<MpvHandle object at 0x7f463ac8ee18>, <mpv.c_char_p_Array_4 object at 0x7f46385367b8>))
```
(I am using Python 3.6)

What would be the correct way of passing a string to this function?

**Comment by McSinyx** [comment]
I took a quick look into [the source code](https://github.com/jaseg/python-mpv/blob/master/mpv.py#L758)  and it seems that `mpv.MPV.command` isn't able to pass the string `'-'`. You can use my PR #70 temporarily fixing this or simply `MPV.command('show_text', 'Hello mpv!')` instead.

**Comment by kpj** [comment]
Great catch, thanks!
For now I'll circumvent the problem by setting `duration` manually.

**Comment by jaseg** [comment]
I think I found the root cause of this. If you look at mpv's [```player/command.c```](https://github.com/mpv-player/mpv/blob/942829463410d0e7ab27b3191205ecfd3c132f2d/player/command.c#L5896) the first parameter of ```show-text``` is defined as an integer argument, and [the handler](https://github.com/mpv-player/mpv/blob/942829463410d0e7ab27b3191205ecfd3c132f2d/player/command.c#L5189) actually uses the default ```osd-duration``` if the parameter is set to a value smaller than zero. I think the mpv manpage is just wrong here.

I'll fix the default value in mpv.py to ```-1``` instead of ```-``` and submit a PR to mpv to fix the manpage.

**Comment by jaseg** [comment]
Fixed in a873fd7 / v0.3.9

**Comment by jaseg** [comment]
Upstream PR https://github.com/mpv-player/mpv/pull/6013
